tests/core/bookings/test_repository.py

```diff
@@ -123,8 +123,8 @@ def test_returns_used_past_event_and_thing_bookings_ordered_by_date_created(self
 
         # Then
         assert len(bookings) == 2
-        assert bookings[0] == past_event_booking
-        assert bookings[1] == thing_booking
+        assert bookings[0] == thing_booking
+        assert bookings[1] == past_event_booking
         assert future_event_booking not in bookings
```

Comment:
Si l'ordre n'est pas prévisible, on devrait comparer des *sets* :past_event_booking}` ?

```suggestion
        assert set(bookings) == {thing_booking, past_event_booking}
```